README.md

```diff
@@ -10,34 +10,36 @@ POST /1/login
   * Body: {user: `<uuid>`, password: "secret"}
   * Returns: 200 + a cookie
 
-GET /1/games
+GET /1/opengames
   * Get a list of available games to join
-  * Returns: {games: [{game: `<uuid>`, opponent: `<uuid>`, name: "death match"}, ...]}
-
-GET /1/game/start?join=`<bool>`&user=`<uuid>`
-  * Initialize a new game from an anonymous user
-  * Optional query param "join" allows instant joining of any available game
-  * Optional query param "user" allows joining as a know user
-  * Returns: {game: `<uuid>`, user: `<uuid>`, status: "waiting"|"started", [state: `<boardState>`, color: "white"|"black"]}
-  * A cookie is also set if a game is anonymously joined
+  * Returns: {games: [{game: `<uuid>`, opponent: `<uuid>`}, ...]}
+
+GET /1/mygames?user=`<uuid>`
+  * Get a list of games where "user" is a player
+  * Required query param "user"
+
+POST /1/game/start
+  * Initialize a new game
+  * Body: {user: `<uuid>`, color=`<color>`}
```

Comment:
Don't we need a game uuid too? I'm assuming you can be in more than one at a time

Reply:
I was imagining the server would make up a game uuid